src/main/java/gil/server/HTTPParser.java

```diff
@@ -54,7 +58,17 @@ public String getMethod(String startLine) {
 
     public String getRequestURI(String startLine) {
         String[] startLineArray = startLine.split(SPACE);
-        String URI = startLineArray[1];
+        int indexOfUriInStartLine = 1;
```

Comment:
I like the fact that you are not inserting random 1's and 0's around your code. For added clarity, it might be good to define this as a constant.

Reply:
Thank you for pointing this out!